src/track/trackinfo.h

```diff
@@ -25,7 +25,6 @@ class TrackInfo final {
     MIXXX_DECL_PROPERTY(QString, encoder, Encoder)
     MIXXX_DECL_PROPERTY(QString, encoderSettings, EncoderSettings)
 #endif // __EXTRA_METADATA__
-    MIXXX_DECL_PROPERTY(QString, genre, Genre)
```

Comment:
Removing this removes the macro created getGenre() and set Genre() functions that are still used I think. 
I think we need the single string presentation or something similar to track what we have found in the track metadata.
The alternative would be a lossless round trip. But this will make later normalization harder. This depends however on the strategy of the usage of multi genre fields which are supported or not depending on the file type.  

Do we have a plan here?